scripts/haddocks.sh

```diff
@@ -0,0 +1,113 @@
+#!/usr/bin/env bash
```

Comment:
If the repo supports `ghc-9.4` you could replace this just with:
```
cabal haddock-project --hackage all
```
One needs `cabal-3.10.1.0` for that.

Reply:
It might be worth switching in the future, but right now I don't want to rely on GHC-9.6, since it barely has been added to CI